### PR TITLE
Improvements in README

### PR DESCRIPTION
**Pull request by snapshotpl**
@localheinz :-)

### CODE REVIEW

**Comment by localheinz** [comment]
@snapshotpl 

Awesome! May I ask you to make another change?

File: README.md [hunk 1]
```diff
@@ -37,6 +34,4 @@ This site will eventually be a community site for publishing and sharing Zend Fr
 
 ## Deployment
 
-The master branch of this repository is automatically deployed live to
-[modules.zendframework.com](http://modules.zendframework.com/) every 10
-minutes.
+The master branch of this repository is automatically deployed live to [modules.zendframework.com](http://modules.zendframework.com/) every 10 minutes.
```
Comment (localheinz) [inline README.md]: The `master` branch is currently manually deployed by @GeeH, we don't have an automated deployment process as of the moment. Hoping to have one soon, though!

Comment (snapshotpl) [inline README.md]: So remove this one?

Comment (localheinz) [inline README.md]: Either remove the whole section, or maybe better, replace with a note that it's manually deployed. What do you think?

**Comment by snapshotpl** [comment]
@localheinz done :trollface:

**Comment by localheinz** [comment]
Thank you, @snapshotpl!

**Pull request merged**